Library/Homebrew/cask/dsl/caveats.rb

```diff
@@ -93,6 +93,17 @@ def eval_caveats(&block)
         EOS
       end
 
+      caveat :unsigned_artifact do
+        <<~EOS
+          #{@cask} has an unsigned binary which may prevent it from running on Apple
```

Comment:
```suggestion
          #{@cask} has an unsigned binary which will prevent it from running on Apple
```

unless I've misunderstood